api/server/publications.ts

```diff
@@ -1,13 +1,17 @@
 import { Meteor } from 'meteor/meteor';
 import { Mongo } from 'meteor/mongo';
-import { Users, Messages, Chats } from '../collections/whatsapp-collections';
 import { User, Message, Chat } from 'api/models/whatsapp-models';
+import { Users, Messages, Chats } from '../collections/whatsapp-collections';
 
 export function initPublications() {
-  Meteor.publish('users', function(): Mongo.Cursor<User> {
+  Meteor.publish('users', function(phoneNumbers?: number[]): Mongo.Cursor<User> {
     if (!this.userId) return;
 
-    return Users.collection.find({}, {
+    const query = !phoneNumbers ? {} : {
+      phone: { $in: phoneNumbers }
+    };
+
+    return Users.collection.find(query, {
       fields: {
         profile: 1
```

Comment:
I have an upcoming PR heavily based on the `profile` filed, removing it would also require me to change the application greatly. Can you accept it in this PR and I will fix it on a separate one? As for now I will just deny any update operations. It will sure ease up on me.